fetch.bs

```diff
@@ -810,6 +810,20 @@ the empty string,
 "<code>worker</code>", or
 "<code>xslt</code>". Unless stated otherwise it is the empty string.
 
+<p>A <dfn export id=concept-potential-destination>potential destination</dfn> is "<code>fetch</code>" or a <a for=request>destination</a>.
+
+<p>To <dfn export for=destination id=concept-potential-destination-translate>translate</dfn> a
+<a for=/>potential destination</a> into a <a for=request>destination</a>, run these steps:
```

Comment:
"To translate a potential destination _potentialDestination_, run these steps:"

Variables need to be introduced and we usually don't specify the return value.